src/app/public/modules/action-button/action-button-container.component.scss

```diff
@@ -1,55 +1,11 @@
 @import "~@skyux/theme/scss/mixins";
 @import "~@skyux/theme/scss/_compat/mixins";
 
-.sky-action-button-container {
-  // Note: this has to go on the element as the flex stretch will consume the margin if not.
-  ::ng-deep sky-action-button {
```

Comment:
I moved these to the action-button SCSS file, using `:host-context` instead.

Reply:
Look ok. Just curious - why did you need to use `:host-context`?

Reply:
`:host-context` is the preferred way to "overwrite" styles from a parent context. Before, these styles were using `::ng-deep` in the action-button-container component (which is deprecated behavior). Instead, they should be in the action-button (the subject of the change) stylesheet, using `:host-context`. See: https://angular.io/guide/component-styles#host-context

Reply:
Thanks for the explanation!